Description: Small typo for the word attempts in src/dock.h.
Author: Rodolfo García Peñas (kix) <kix@debian.org>
Forwarded: http://lists.windowmaker.org/dev/msg05381.html
Last-Update: 2013-09-09
--- a/src/dock.c
+++ b/src/dock.c
@@ -4284,7 +4284,7 @@
 	}
 
 	if (i == UNIQUE_NAME_WATCHDOG)
-		wwarning("Couldn't find a unique name for drawer in %d attemps.", i);
+		wwarning("Couldn't find a unique name for drawer in %d attempts.", i);
 #undef UNIQUE_NAME_WATCHDOG
 
 	return buffer;
